### PR TITLE
Fix undefined symbols error for iOS.

### PR DESCRIPTION
**Pull request by jianjunz**
Some features are only available on Windows.

### CODE REVIEW

File: talk/owt/sdk/include/cpp/owt/base/stream.h [hunk 1]
```diff
@@ -99,9 +92,16 @@ class Stream {
   */
   virtual StreamSourceInfo Source() const;
 #if defined(WEBRTC_WIN)
```
Comment (taste1981) [inline talk/owt/sdk/include/cpp/owt/base/stream.h]: applies to Linux as well.

File: talk/owt/sdk/base/stream.cc [hunk 2]
```diff
@@ -150,6 +150,7 @@ void Stream::SetAudioTracksEnabled(bool enabled) {
   }
 }
 
+#if defined(WEBRTC_WIN)
```
Comment (taste1981) [inline talk/owt/sdk/base/stream.cc]: this applies to WEBRTC_LINUX as well.

File: talk/owt/sdk/include/cpp/owt/base/stream.h (hunk 1 above)
Comment (jianjunz) [reply talk/owt/sdk/include/cpp/owt/base/stream.h]: Fixed.

File: talk/owt/sdk/base/stream.cc (hunk 2 above)
Comment (jianjunz) [reply talk/owt/sdk/base/stream.cc]: Fixed.

**Pull request merged**